test/services/user/unit.test.js

```diff
@@ -226,10 +226,11 @@ describe('publicTeachers service', () => {
 		});
 		// reset TEACHER_VISIBILITY_FOR_EXTERNAL_TEAM_INVITATION back to original value
 		// eslint-disable-next-line max-len
-		app.Config.set('TEACHER_VISIBILITY_FOR_EXTERNAL_TEAM_INVITATION', ORIGINAL_TEACHER_VISIBILITY_FOR_EXTERNAL_TEAM_INVITATION);
+		Configuration.set('TEACHER_VISIBILITY_FOR_EXTERNAL_TEAM_INVITATION', ORIGINAL_TEACHER_VISIBILITY_FOR_EXTERNAL_TEAM_INVITATION);
 	});
 
 	after(async () => {
 		await testObjects.cleanup();
 	});
 });
+ */
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4991925)